cmake_test/asserts/equal.cmake

```diff
@@ -13,8 +13,8 @@ include_guard()
 #]]
 function(ct_assert_equal _ae_var _ae_contents)
```

Comment:
CMakePPCore also has asserts. It'd be good to keep them all in one place. I recommend moving these asserts there.